### ISSUE TITLE
collections.items.getItem prints spurious warning

### ISSUE THREAD

**Issue by rjrodger** [webflow/js-webflow-api#128]
Calling `WebflowClient.collections.items.getItem(...)` returns correct item, but prints:

```
Failed to validate.
  - response.fields: Expected object. Received list.
```

**Comment by dsinghvi** [comment]
@rjrodger thanks for reporting this issue. We are pushing an update to the SDK shortly!

**Comment by dsinghvi** [comment]
@rjrodger this is now released in `v2.1.2`!

**Comment by rjrodger** [comment]
amazing! thank you so much!
